hw/ip/otp_ctrl/data/otp_ctrl_mmap.hjson

```diff
@@ -176,6 +176,14 @@
                     name: "CREATOR_SW_CFG_DEFAULT_BOOT_DATA_IN_PROD_EN",
                     size: "4"
                 }
+                {
+                    name: "CREATOR_SW_CFG_RMA_SPIN_EN",
+                    size: "4"
+                }
+                {
+                    name: "CREATOR_SW_CFG_RMA_SPIN_CYCLES",
```

Comment:
just to double check that I'm remembering correctly, do we think this is enough?
If we were to max out this value, it's something like...40 seconds, so that would be the timeout of a user needing to initiate life cycle activity right?

it seems sufficient to me.

Reply:
Thanks for checking @tjaychen. Yes, assuming that the CPU is at 100 MHz, which depends on the calibration I guess, this would be ~42 seconds. We can revisit this part later if you think it won't be enough.